v4/source/definitions.yaml

```diff
@@ -578,9 +578,6 @@ components:
         auth_service:
           description: The authentication service such as "email", "gitlab", or "ldap"
           type: string
```

Comment:
Can we mark the auth_data and auth_service fields as required

Reply:
Sure, good one.